syft/frameworks/torch/tensors/interpreters/hook.py

```diff
@@ -8,7 +8,9 @@ class HookedTensor(AbstractTensor):
     method hooked without having to add it to the hook.py file.
     """
 
-    def __init__(self, owner=None, id=None, tags=None, description=None, verbose=False):
+    def __init__(
+        self, numpy_tensor=None, owner=None, id=None, tags=None, description=None, verbose=False
```

Comment:
I think we actually want to make the reverse change here; the Numpy code got moved over to [NumpyTensor](https://github.com/OpenMined/PySyft/blob/be9e0d894e7fe09cf22f93a3a27ad42a3903a037/syft/frameworks/torch/tensors/interpreters/numpy.py), so we should probably remove both the `numpy_tensor` parameter and the docstring explanation of it.

Reply:
oh okay. fixed it now.